mobly/records.py

```diff
@@ -468,8 +463,8 @@ def add_error(self, position, e):
 
   def __str__(self):
     d = self.to_dict()
-    l = ['%s = %s' % (k, v) for k, v in d.items()]
-    s = ', '.join(l)
+    v = ['%s = %s' % (k, v) for k, v in d.items()]
```

Comment:
Using variable name `l` is against flake8 rule [`E741`](https://www.flake8rules.com/rules/E741.html):
> Variables named `I`, `O`, and `l` can be very hard to read. This is because the letter `I` and the letter `l` are easily confused, and the letter `O` and the number `0` can be easily confused.

I can ignore this warning by update `.flake8` setting. Any advice is appreciated.

Reply:
I see, we can give it a more descriptive name, like "kv_pairs"

Reply:
Many thanks for the suggestion. Done in below commit:
https://github.com/google/mobly/pull/742/commits/b104346d399e4195dd64216794d767d6061da2a7